File: Praktikum/Praktikum 1/BunchOfKeys.cpp
#include "BunchOfKeys.hpp"
#include <iostream>

BunchOfKeys :: BunchOfKeys(): n_keys(0){}
void BunchOfKeys :: add(){
    n_keys +=1;
}
void BunchOfKeys :: shake(){
    if(n_keys>1){
        for(int i=0; i<n_keys; i++){
            cout << "krincing" << endl;
        }
    }else{
        cout << "Tidak terjadi apa-apa" << endl;
    }
}

File: Praktikum/Praktikum 4/1/BunchOfKeys.java
public class BunchOfKeys{
    private int n_keys;

    public BunchOfKeys(){
        this.n_keys = 0;
    }

    public int getNKeys(){
        return n_keys;
    }

    public void add(){
        this.n_keys +=1;
    }

    public void shake(){
        if(this.n_keys<=1){
            System.out.println("Tidak terjadi apa-apa");
        }else{
            int i;
            for(i=0; i<this.n_keys; i++){
                System.out.println("krincing");
            }
        }
    }
}